### PR TITLE
Remove adsk references

### PR DESCRIPTION
**Pull request by ruiterr**
This PR removes most of the remaining ADSK namespace references, fixes linting and fixes a few bugs in the examples.

There are a few left:
* index.d.ts still refers to workspace
   Is this file up-to-date? Should we replace this workspace reference with a different type? Or do we have to regenerate the file (it seems it is autogenerated via tsd-jsdoc)
* data-binder referes to an HFDMWorkspaceComponent. Why is there a reference to a component? I assume we don't use components? So should this be removed.
* There are references in the karama config files. Are those used at all any longer?
* We have two tests usd and performance, which are not run, but still contain adsk references. Are those worth keeping? Ar they worth fixing?

For inspector table, I removed the lint command for the momet. It was using tslint, but the corresponding config file was missing. We probably should switch to the eslint settings from the project, but for this, we first have to fix the linter errors.

@evaliyev @nedalhy Can you comment on the questions above? I you review it, best disable whistspaces since I auto-fixed the linting.

@DLehenbauer CC

### CODE REVIEW

File: experimental/PropertyDDS/packages/property-inspector-table/webpack.appfw.base.js [hunk 1]
```diff
@@ -51,9 +51,7 @@ module.exports.TSConfig = function(fileType, generateDeclarations, distTypesPath
         rules: [
           {
```
Comment (nedalhy) [inline experimental/PropertyDDS/packages/property-inspector-table/webpack.appfw.base.js:52]: Probably we should rename this file

**Review by nedalhy — APPROVED** [review]
LGTM, i tried to go quickly through the linting changes, hopefully I didn't overlook a code change.

File: experimental/PropertyDDS/packages/property-binder/index.d.ts [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
  *    pick up definitions from this file.
  * 2. By wrapping everything in the 'declare module' statement, all of the 'declare' statements had to change to
  *    'export'.
  * 3. tsd-jsdoc appears to not handle object parameters correctly, adding them as
  *    (params: any, params.arg0: any, ..., params.argn: any), which is a syntax error in typescript.
  */
 declare module "@fluid-experimental/property-binder" {
-    import { BaseProperty, ContainerProperty, ValueProperty, Workspace } from '@adsk/forge-hfdm';
+    import { BaseProperty, ContainerProperty, ValueProperty} from '@fluid-experimental/property-properties';
+    import { Workspace } from '@adsk/forge-hfdm';
```
Comment (evaliyev) [inline experimental/PropertyDDS/packages/property-binder/index.d.ts:16]: We will fix it later, I guess :)

File: experimental/PropertyDDS/packages/property-binder/package.json [hunk 3]
```diff
@@ -42,6 +42,7 @@
     "@babel/preset-env": "^7.2.0",
     "@fluidframework/build-common": "^0.21.0-0",
     "@fluidframework/test-utils": "^0.39.0",
+    "@fluidframework/test-runtime-utils": "^0.39.0",
```
Comment (evaliyev) [inline experimental/PropertyDDS/packages/property-binder/package.json:45]: Is it used anywhere?

**Review by evaliyev — APPROVED** [review]

File: experimental/PropertyDDS/packages/property-inspector-table/webpack.appfw.base.js (hunk 1 above)
Comment (ruiterr) [reply experimental/PropertyDDS/packages/property-inspector-table/webpack.appfw.base.js:52]: good point. I'll do that in the other PR.

File: experimental/PropertyDDS/packages/property-binder/index.d.ts (hunk 2 above)
Comment (ruiterr) [reply experimental/PropertyDDS/packages/property-binder/index.d.ts:16]: Yes, I have a second PR open, which changes this.

File: experimental/PropertyDDS/packages/property-binder/package.json (hunk 3 above)
Comment (ruiterr) [reply experimental/PropertyDDS/packages/property-binder/package.json:45]: it is used in `experimental/PropertyDDS/packages/property-binder/test/data_binder/mock_shared_property_tree.ts`, which imports MockFluidDataStoreRuntime from this package.

**Pull request merged**